spec/libsass-closed-issues/issue_759.hrx

```diff
@@ -21,3 +28,49 @@ foo {
   d: 40px;
   e: 50px;
 }
+
+<===> warning
+DEPRECATION WARNING: As of Dart Sass 2.0.0, !global assignments won't be able to
+declare new variables. Consider adding `$a: null` at the top level.
+
+  ,
+1 | $a: 10px !global !default;
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+    /sass/spec/libsass-issues/issue_759/input.scss 1:1  root stylesheet
+
+DEPRECATION WARNING: As of Dart Sass 2.0.0, !global assignments won't be able to
+declare new variables. Consider adding `$b: null` at the top level.
+
+  ,
+2 | $b: 20px !default !global;
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+    /sass/spec/libsass-issues/issue_759/input.scss 2:1  root stylesheet
+
+DEPRECATION WARNING: As of Dart Sass 2.0.0, !global assignments won't be able to
+declare new variables. Consider adding `$c: null` at the top level.
+
+  ,
+3 | $c: 30px !default !default !default !global !global !global;
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Is there an issue for allowing `!default` or `!global` at most once? I believe it was mentioned in the written spec you added recently.

Reply:
Now there is! https://github.com/sass/sass/issues/2607